### ISSUE TITLE
subsubsection not shaded in tableofcontents

### ISSUE THREAD

**Issue by josephwright** [josephwright/beamer#94]
Hi!

I am preparing a presentation that needs to be split in two major parts, so I ended up using `\subsubsections` for the first time in beamer (3.10 on Gentoo). However when showing the section outline in `\AtBeginSection[]` I saw that the sections and subsections would be faded for the non-current sections, but the subsubsections would not (cf. attachment for a minimal example).

I guess this not only not so pretty, but also unintended? ;-)

Best regards!

Hek

````
\documentclass{beamer}
\usetheme{default}

\begin{document}
\AtBeginSection[]
{
  \begin{frame}<beamer>
    \tableofcontents[currentsection]
  \end{frame}
}

\section{A}
\subsection{1}
\subsubsection{i}
\begin{frame}
    A content
\end{frame}

\section{B}
\subsection{1}
\subsubsection{i}
\begin{frame}
    B content
\end{frame}

\end{document}
````

----------------------------------------
- Bitbucket: https://bitbucket.org/rivanvx/beamer/issue/94
- Originally reported by: HekMek
- Originally created at: 2010-11-05

**Comment by josephwright** [comment]
_From Vedran Miletić on 2011-01-03_

Till says in the manual that subsubsections shouldn't be used, but since they are here, we could make them work.

**Comment by josephwright** [comment]
_From Anonymous on 2011-11-23_

Adding `subsubsectionstyle=show/shaded,` to the `\tableofcontents[]` command made the subsubsection titles shaded, i.e.:
````
    \tableofcontents[
    currentsection,
    currentsubsection,
    subsubsectionstyle=show/shaded, 
    ]
````
(Not my idea, I just modified something suggested here: http://tex.stackexchange.com/questions/28654/beamer-table-of-contents-display-all-subsections-below-section)

**Comment by josephwright** [comment]
_From Anonymous on 2012-01-20_

yes, it works, thanks a lot :)

**Comment by josephwright** [comment]
Fixed by 17ef800959f26f589d2086705d099b4fd933c3da
